packages/labs/analyzer/src/lib/javascript/classes.ts

```diff
@@ -95,6 +96,20 @@ export const getClassMembers = (
         })
       );
     } else if (ts.isPropertyDeclaration(node)) {
+      if (!ts.isIdentifier(node.name)) {
+        analyzer.diagnostics.push(
```

Comment:
This could have a lot of false-positives, for well-known symbols, private fields, or overrides of protected methods (ie, you don't want to see a warning for a method you _had_ to name this way). Could we check for those cases and for the rest have a separate code so it's more easily disabled?